controller/lib/hal/adc.cpp

```diff
@@ -168,32 +193,74 @@ void HalApi::InitADC() {
   adc->adc[0].cfg1.dmacfg = 1;
   adc->adc[0].cfg1.cont = 1;
 
-  // Enable ADC oversampling.
+  // [RM] 16.6.5: ADC configuration register 2 (ADC_CFGR2) (pg 462)
+  /*
+  Enable ADC oversampling: Write 1 to bit 0
+  Bit 0 ROVSE: Regular Oversampling Enable
+    0: Regular Oversampling disabled
+    1: Regular Oversampling enabled
+  */
```

Comment:
I see what you mean. Since I went through this file before the registers file, I added a lot of info about registers here which better belongs with the register definitions. Will clean up where this happened elsewhere